service_thing_aggregate/app/repositories/dao/ObjectDao.scala

```diff
@@ -481,6 +481,33 @@ class ObjectDao @Inject()(
       }
   }
 
+  def findByUUID(
+    museumId: MuseumId,
+    objectUUID: ObjectUUID,
+    collections: Seq[MuseumCollection]
+  )(implicit currUsr: AuthenticatedUser): Future[MusitResult[Option[MusitObject]]] = {
+    val cids = collections.flatMap(_.schemaIds).distinct
+
+    val q1 = objTable.filter { o =>
```

Comment:
By renaming this variable to `queryAllCollecitons` the comment at line 496 might be redundant.